x-pack/qa/repository-old-versions/src/test/java/org/elasticsearch/oldrepos/OldRepositoryAccessIT.java

```diff
@@ -130,10 +162,98 @@ public void testOldRepoAccess() throws IOException {
                 assertEquals(0, snapshotStatus.getShardsStats().getFailedShards());
                 assertThat(snapshotStatus.getStats().getTotalSize(), greaterThan(0L));
                 assertThat(snapshotStatus.getStats().getTotalFileCount(), greaterThan(0));
+
+                // so far only added basic infrastructure for reading 6.0.0+ indices
+                if (Build.CURRENT.isSnapshot() && oldVersion.onOrAfter(Version.fromString("6.0.0"))) {
```

Comment:
At some point we could reintroduce the Version constants for old Elasticsearch versions.

Reply:
This logic will completely change in the follow-up.